conf/application.conf

```diff
@@ -133,6 +133,9 @@ feature-switch {
   enable-calc-breakdown = false
   enable-calc-dataApi = true
   enable-obligations-page = false
```

Comment:
can you please also delete the below FS which are not in use anymore but we have never updated.

  enable-account-details = true
  enable-calc-breakdown = false
  enable-calc-dataApi = true

Reply:
Done